lgc/include/lgc/state/PalMetadata.h

```diff
@@ -171,6 +171,14 @@ class PalMetadata {
   // Set userDataLimit to maximum
   void setUserDataLimit();
 
+  // Returns true of the some of the user data nodes are spilled.
+  bool userDataNodesAreSpilled() const { return m_spillThreshold->getUInt() != USER_DATA_SPILL_IS_UNUSED; }
+
+  // Returns true of the setting in the PAL meta data require all of the user data nodes.
+  bool pipelineRequiresAllUserData() const;
+
+  const uint64_t USER_DATA_SPILL_IS_UNUSED = UINT_MAX;
```

Comment:
I'm OK with this. Not an issue.

As I know, user data limit in PAL is expected to be a 16-bit integer with the 0xFFFF as the upbound. But LLPC uses 32-bit limit for a long time.